org.kde.rkward.json

```diff
@@ -0,0 +1,74 @@
+{
+    "id": "org.kde.rkward",
+    "branch": "master",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "base": "io.qt.qtwebengine.BaseApp",
+    "base-version": "5.15-21.08",
+    "command": "rkward",
+    "tags": ["nightly"],
+    "desktop-file-name-suffix": " (Nightly)",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--share=network",
+        "--filesystem=host"
```

Comment:
I don't think this will grant you access to what you want. You won't have access to whatever the distro installs.

Reply:
Sigh... I see what you mean (I had tested that I was able to load packages outside the app filesystem, but not compiling new packages.)

I have a plan (of sorts) for working around this, but it won't be something to patch in a few minutes and/or lines, so putting this on hold for now ([WIP] tag added).